src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -2,20 +2,41 @@
 
 import java.sql.Connection;
 import java.sql.DriverManager;
+import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.sql.Statement;
 
 public class DatabaseManager {
     private static final String DB_CONNECTION_URL = "jdbc:postgresql://127.0.0.1:5432/";
 
     private Connection connection;
 
-    boolean connect(String database, String user, String password) {
+    public boolean connect(String database, String user, String password) {
         try {
             Class.forName("org.postgresql.Driver");
-            connection = DriverManager.getConnection(DB_CONNECTION_URL + database, user, password);
+            connection = DriverManager.getConnection(DB_CONNECTION_URL + database+"?loggerLevel=OFF", user, password);
```

Comment:
It's an interesting feature, but I would extracted this parameter

Reply:
I'm not sure that there's any need to extract this parameter for few reasons:
1. loggerLevel=OFF - this string tells everything about itself;
2. This parameter is local, it's just a property of JDBC driver to disable console logging from postgres and I'm not planning to use it in any other places, but if I will - than for sure I will extract it.
Thanks.